.github/workflows/scripts/get_dependencies.sh

```diff
@@ -124,6 +124,12 @@ get_gcovr() {
   pip install gcovr
 }
 
+# Wraps instally cppyy
+get_cppyy() {
+  sudo apt update
+  pip3 install cppyy
```

Comment:
Based on gcovr we were just using `pip` not `pip3` to install. We should stay consistent (probably worth declaring it as a variable like `PIP_COMMAND`).